client.go

```diff
@@ -412,6 +412,7 @@ func (c *Client) send(verb string, p string, body io.Reader) (io.ReadCloser, int
 	}
 
 	req.Header.Add("Authorization", "Bearer "+c.apiKey)
+	req.Header.Set("Content-Type", "application/json")
```

Comment:
If Jerry Foster says it, then I like it.

Reply:
this is very true!